pkg/fakerp/server.go

```diff
@@ -265,12 +264,10 @@ func (s *Server) handlePut(w http.ResponseWriter, req *http.Request) {
 	}
 
 	// simulate Context with property bag
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Hour)
+	// TODO: Populate context from request header
+	ctx, cancel := context.WithTimeout(context.Background(), s.timeout)
```

Comment:
Not much gain from making this configurable too

Reply:
just consistency, I think it just makes the behavior more understandable. but agreed, not much value

Reply:
this has gone in the wrong direction - why can't we get rid of the timeout code across the board?